ui/src/pages/backoffice/Patron/PatronDetails/components/PatronActionMenu/PatronActionMenu.js

```diff
@@ -13,11 +13,19 @@ export default class PatronActionMenu extends Component {
           <ScrollingMenuItem label="Checkout" elementId="patron-checkout" />
           <ScrollingMenuItem label="Current loans" elementId="current-loans" />
           <ScrollingMenuItem label="Loan requests" elementId="loan-requests" />
+          <ScrollingMenuItem
+            label="Borrowing requests"
```

Comment:
Can you update all the menu to reflect the titles? They should be similar.
You can shorten the titles of the boxes, no need for `Patron's...`